packages/dashboard/src/components/pageStructure/index.js

```diff
@@ -190,35 +242,48 @@ export function LeftRail() {
         </Content>
         <Content>
           <NavList>
-            {enabledPrimaryPaths.map(({ Icon, ...path }) => (
-              <NavListItem key={path.value}>
-                <NavLink
-                  active={path.value === state.currentPath}
-                  href={resolveRoute(path.value)}
-                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
-                  isBold
-                  aria-label={
-                    path.value === state.currentPath
-                      ? sprintf(
-                          /* translators: %s: the current page, for example "My Stories". */
-                          __('%s (active view)', 'web-stories'),
-                          path.label
-                        )
-                      : path.label
-                  }
-                  {...(path.isExternal && {
-                    rel: 'noreferrer',
-                    target: '_blank',
-                    onClick: (evt) => onExternalLinkClick(evt, path),
-                  })}
-                >
-                  {Icon && <Icon width="22px" />}
-                  <PathName as="span" isBold>
-                    {path.label}
-                  </PathName>
-                </NavLink>
-              </NavListItem>
-            ))}
+            {enabledPrimaryPaths.map(({ Icon, ...path }) => {
+              const isNotificationBubbleEnabled =
+                path.value === APP_ROUTES.TEMPLATES_GALLERY;
+              return (
+                <NavListItem key={path.value}>
+                  <NavLink
+                    active={path.value === state.currentPath}
+                    href={resolveRoute(path.value)}
+                    size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                    isBold
+                    aria-label={
```

Comment:
🤔  should the new templates be included in the aria label like `explore templates (48 new)` ?

Reply:
good call